pyiron_lammps/compatibility/calculate.py

```
@@ -0,0 +1,412 @@
+        style ('cg'/'sd'/other values from Lammps docs): The style of the numeric minimization, either conjugate
+            gradient, steepest descent, or other keys permissible from the Lammps docs on 'min_style'. (Default
+            is 'cg' -- conjugate gradient.)
+        rotation_matrix (numpy.ndarray): The rotation matrix from the pyiron to Lammps coordinate frame.
+    """
+    # This docstring is a source for the calc_minimize method in pyiron_atomistics.lammps.base.LammpsBase.calc_minimize and
+    # pyiron_atomistics.lammps.interactive.LammpsInteractive.calc_minimize -- Please ensure that changes to signature or
+    # defaults stay consistent!
+
+    max_evaluations = 100 * max_iter
+    if n_print > max_iter:
+        warnings.warn("n_print larger than max_iter, adjusting to n_print=max_iter")
```

Ruff (0.14.5)

No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)